libs/core/src/components/pds-button/docs/pds-button.mdx

```diff
@@ -3,7 +3,7 @@ import { components } from '../../../../dist/docs.json';
 
 # Button
 
-A button is an interactive element that can be clicked to perform an action or submit a form.
+A button is an interactive element that can be clicked to perform various actions such as triggering an event, submitting data, or navigating to a different location.
```

Comment:
Just a thought, but can we separate the "navigating to a different location" under a sub-section called "Call-To-Action (CTA) Button" with a brief description? This could assist with how and when to use.

Reply:
There is an entire section for CTA that includes that. 

> This is useful for creating call-to-action buttons that navigate to external resources or different parts of your application.

I don't know that is needs to be excluded from the general overall button component description, but should be clarified in the section docs.

Reply:
When you put in the form of "general" description, then this makes sense. Thanks for clarifying.